src/main/java/de/lukweb/justmail/socket/ImapSocketHandler.java

```diff
@@ -65,16 +65,16 @@ public void run() {
                 }
                 cache += (char) read;
                 if (!(cache.endsWith("\n\r") || cache.endsWith("\n"))) continue;
-                /* if (session.getCallback() != null) {
+                if (session.getCallback() != null) {
```

Comment:
Here we have to create a abstract class for both handlers, because this code is twice, but it isn't your problem. I will do this in the future. :)